action.yml

```diff
@@ -0,0 +1,28 @@
+name: "vulnbot-action"
+description: ""
+inputs:
+  config:
+    default: config.toml
+    description: "Path that specifies the ecosystems for which the VulnBot will be reporting vulnerabilities."
+    required: false
+  slack_auth_token:
+    description: "Represents the Slack authentication token that allows the VulnBot to access and post messages to a Slack channel."
+    required: false
+  github_token:
+    description: "Access token that allows the VulnBot to interact with the GitHub API."
+    required: false
+  github_org:
+    description: "Refers to the GitHub organization that the repository belongs to."
+    required: false
+runs:
+  using: docker
+  image: Dockerfile
```

Comment:
[q] Is it possible to just use the upstream Docker image directly, rather than having our intermediate one in this repo? It seems the one here, and the entrypoint, are not really doing much of anything. But I'm not sure if they're fulfilling some requirement that I'm missing.